services/galley/test/integration/API/MLS/Util.hs

```diff
@@ -890,3 +893,59 @@ receiveNewRemoteConv conv gid = do
       client
       (qDomain conv)
       nrc
+
+receiveOnConvUpdated ::
+  (MonadReader TestSetup m, MonadIO m) =>
+  Qualified ConvId ->
+  Qualified UserId ->
+  Qualified UserId ->
+  m ()
+receiveOnConvUpdated conv origUser joiner = do
+  client <- view tsFedGalleyClient
+  now <- liftIO getCurrentTime
+  let cu =
+        ConversationUpdate
+          { cuTime = now,
+            cuOrigUserId = origUser,
+            cuConvId = qUnqualified conv,
+            cuAlreadyPresentUsers = [qUnqualified joiner],
+            cuAction =
+              SomeConversationAction
+                SConversationJoinTag
+                ConversationJoin
+                  { cjUsers = pure joiner,
+                    cjRole = roleNameWireMember
+                  }
+          }
+  void $
+    runFedClient
+      @"on-conversation-updated"
+      client
+      (qDomain conv)
+      cu
+
+getGroupInfo ::
+  ( HasCallStack,
+    MonadIO m,
+    MonadCatch m,
+    MonadThrow m,
+    MonadHttp m,
+    HasGalley m
+  ) =>
+  UserId ->
+  Qualified ConvId ->
+  m ResponseLBS
+getGroupInfo sender qcnv = do
+  galley <- viewGalley
+  get
+    ( galley
+        . paths
+          [ "v2",
```

Comment:
If you rebase, you won't need the v2 prefix.

Reply:
Thanks! Fixed.